x-pack/test/api_integration/apis/monitoring_collection/index.ts

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+export default function ({ loadTestFile }: FtrProviderContext) {
+  describe('Monitoring Collection', function taskManagerSuite() {
+    loadTestFile(require.resolve('./prometheus'));
```

Comment:
I think you might have forgotten to push the change to `x-pack/test/api_integration/apis/index.ts` to include the test

Reply:
good catch. thanks